resources/sdk/clisdkclient/templates/restclient.mustache

```diff
@@ -243,6 +269,48 @@ func authorize(c config.Configuration) (models.OAuthTokenData, error) {
 	return *oAuthTokenResponse, nil
 }
 
+func startCodeAuthServer(wg *sync.WaitGroup, uri string) {
+	address := strings.Replace(uri, "http://localhost", "", -1)
+	port := address[0:5]
+	path := strings.Replace(address, port, "", -1)
+	authCode := ""
+
+	getAuthCode := func(w http.ResponseWriter, r *http.Request) {
+		authCode = r.URL.Query().Get("code")
+		if authCode != "" {
+			codeAuthCode = authCode
+			wg.Done()
+			return
+		}
+	}
+
+	http.HandleFunc(path, getAuthCode)
+	log.Fatal(http.ListenAndServe(port, nil))
+}
+
+func openBrowserForLogin(wg *sync.WaitGroup, clientID string, redirectURI string) {
+	encodedURI := strings.Replace(redirectURI, ":", "%3A", -1)
+	encodedURI = strings.Replace(redirectURI, "/", "%2F", -1)
+	loginURL := fmt.Sprintf("https://login.mypurecloud.com/oauth/authorize?client_id=%v&response_type=code&redirect_uri=%v", clientID, encodedURI)
```

Comment:
The login URL wouldn't be `mypurecloud.com` for everyone. You can use the `Environment()` from the config to replace it